index.css

```diff
@@ -0,0 +1,221 @@
+html, body {
```

Comment:
Good work. CSS is definitely on point with page displaying in column depending on window size. Some minor feedback would be to use "border-radius" for rounding up login input boxes, other than that good job with emulating blue bird in the background with footer at the bottom. Next time try to use color picker extension on Chrome to get colors that are identical to Twitter page. Keep it up!